ci2/ci/templates/pr.html

```diff
@@ -23,35 +23,95 @@ <h2>Batch {{ batch['id'] }} Jobs</h2>
         </tr>
       </thead>
       <tbody>
-	{% for job in batch['jobs'] %}
+	      {% for job in batch['jobs'] %}
         <tr>
           <td align="right">{{ job['id'] }}</td>
           <td align="left">{{ job['attributes']['name'] }}</td>
           <td align="left">{{ job['state'] }}</td>
           <td align="right">
-	    {% if 'exit_code' in job %}
-	    {{ job['exit_code'] }}
-	    {% endif %}
-	  </td>
+	          {% if 'exit_code' in job %}
+	          {{ job['exit_code'] }}
+	          {% endif %}
+	        </td>
           <td align="right">
             {% if 'duration' in job and job['duration'] %}
             {{ job['duration'] }}
             {% endif %}
           </td>
           <td align="left">
-	    <a href="/jobs/{{ job['id'] }}/log">log</a>
-	  </td>
+	          <a href="/jobs/{{ job['id'] }}/log">log</a>
+	        </td>
           <td>
             {% if 'link' in job['attributes'] %}
             {% for name in job['attributes']['link'] %}
             <a href="https://{{ name }}.{{ job['attributes']['domain'] }}">{{ name }}</a>
-	    {% endfor %}
+	          {% endfor %}
             {% endif %}
           </td>
         </tr>
-	{% endfor %}
+	      {% endfor %}
       </tbody>
       {% endif %}
     </table>
+    {% if most_recent_build is not none }
+    <h2>Most Recent Complete Build: Batch {{ most_recent_build.batch['id'] }} Jobs</h2>
```

Comment:
Complete seems wrong here since it might still be in progress.  Minor UI thing, but I'd vote against having both the current and most recent builds in full on this page.  I also think the current build shouldn't be here, just links to the current and most recent builds as well as the full history.  Thoughts?

Reply:
I'm gonna remove all of this and just rely on the history PR to display this information.